Controller.java

```diff
@@ -125,7 +125,91 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 				request.setAttribute("title", title);
 				request.getRequestDispatcher("ViewArticle.jsp").forward(request, response);
 			}		
-		}		
+		}
+		
+		
+		if(page.equals("comments_like"))
+		{
+			String title = request.getParameter("title");
+			String ip = request.getRemoteAddr();
+			int id = Integer.parseInt(request.getParameter("id"));
+			
+//			System.out.println(id);
+//			System.out.println(title);
+			
+			int a = 1;
+			
+			DB db = new DB();
+			boolean status = false;
+			
+			try
+			{
+				status = db.check_comments_popular(ip, id, a);	

```

Comment:
@mjhmohamed @hema-001 

What is the purpose of this statement?  Why was `a` set 1?

Hui

Reply:
`status` is a flag to check whether the user has already liked the comment or not.
the function `check_comments_popular(ip, id, a);` is going to check if this comment has been liked by this user or not, if he did like it, then a pop-up message will remind him that he did so. Else, a new like will be added to the like count of the comment.